frontend/src/assets/data/AreaCodes.json

```diff
@@ -0,0 +1,1492 @@
+[
+  {
```

Comment:
Wow this is cool! How did you generate this?

Reply:
https://github.com/ravisorg/Area-Code-Geolocation-Database/blob/master/us-area-code-geo.csv then https://csvjson.com/csv2json to convert, working on how to utilize it